capstone/README.md

```diff
@@ -69,16 +45,17 @@ Make sure you have the following installed:
 Here's whats next for our project
```

Comment:
Unclear if a roadmap should include an overview of the dev process including where we started, where we are now and what is next. Let's discuss this in standup.